analyses/tcga-capture-kit-investigation/README.md

```diff
@@ -41,10 +41,9 @@ We created script to prepare the input dataframe for the boxplot script. This sc
 - output: `plots/boxplot-*.png`
 ![](plots/boxplot-strelka2.png)
 
-### (WIP) coverage-comparison
-| sample name                  | # of bases > 20x vs GDC BED | % of bases > 20x vs GDC BED | # of bases > 20x vs MC3 BED | % of bases > 20x vs MC3 BED | 
-|------------------------------|------------------------------|------------------------------|-----------------------------|-----------------------------| 
-| TCGA-HT-7684-01A-11D-2253-08 | 27755836                     | 84%                          | 8787516                     | 31%                         | 
-| TCGA-SP-A6QF-01A-12D-A35I-08 | 27350549                     | 82%                          | 30060356                    | 31%                         | 
+### Coverage-Comparison
+In order to show in the form of coverage that the downloaded BED files were  correct, we calculated the number of bases that are covered  at atleast 20x in both old TCGA BED file(`gencode.v19.basic.exome`) and the new downloaded BED files (`results/*Gh38.bed`). The coverage table in `results/TCGA_oldandnew_coverage_comparisons.txt` gives  a  tabular summary of  number of bases and percentages within corresponding BED files. Boxplot comparison for the same data is available here(`plots/TCGA_oldandnew_coverage_plots.png`)
```

Comment:
Did you calculated the coverage based on the actual BED for each samples? how did you handle those multiple-capture-kit `|` delimited samples?

also add a space for "atleast"?

Reply:
These were submitted yesterday before we had the file with two capture kit BED's for the 5 samples. I am doing the BED intersection now, I can re-submit the 5 samples and update. these results.

Reply:
As per our discussion, since I am using the larger BED file to calculate these coverages here, we are going to leave these for now.